manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -244,6 +245,7 @@ vipLoop:
 		}
 		for _, nAttach := range specNetworks {
 			if nAttach.Target == eAttach.NetworkID {
+				log.L.WithFields(logrus.Fields{"service_id": s.ID, "vip": eAttach.Addr}).Infof("allocate vip")
```

Comment:
super nit: use Info instead fo Infof ?

Reply:
I think this should be Debug ? Info will print it for every ip ?

Reply:
I was actually thinking to want this printed. Concerns?

Reply:
@fcrisciani only concern is that it might be too much to print in the logs. We normally would like to avoid that.

Reply:
Definitely change from `Info` to `Debug`. It'll be too verbose for `Info`.

Reply:
@nishanttotla today we have 0 visibility and debug is very difficult, do you have any other idea?